src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/LegacyPropagator.cs

```diff
@@ -0,0 +1,190 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net;
+using System.Collections.Generic;
+
+namespace System.Diagnostics
+{
+    internal class LegacyPropagator : TextMapPropagator
```

Comment:
```suggestion
    internal sealed class LegacyPropagator : TextMapPropagator
```

Reply:
why this is matter for internal classes?

Reply:
https://github.com/dotnet/runtime/issues/49944